src/routes/organizations/organizations.controller.ts

```diff
@@ -1,6 +1,136 @@
-import { ApiTags } from '@nestjs/swagger';
-import { Controller } from '@nestjs/common';
-
+import {
+  ApiForbiddenResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiResponse,
+  ApiTags,
+  ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  HttpStatus,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+import { OrganizationsService } from '@/routes/organizations/organizations.service';
+import { AuthGuard } from '@/routes/auth/guards/auth.guard';
+import { Auth } from '@/routes/auth/decorators/auth.decorator';
+import { AuthPayload } from '@/domain/auth/entities/auth-payload.entity';
+import { OrganizationStatus } from '@/domain/organizations/entities/organization.entity';
+import { CreateOrganizationResponse } from '@/routes/organizations/entities/create-organizations.dto.entity';
+import {
+  CreateOrganizationDto,
+  CreateOrganizationSchema,
+} from '@/routes/organizations/entities/create-organization.dto.entity';
+import { GetOrganizationResponse } from '@/routes/organizations/entities/get-organization.dto.entity';
+import {
+  UpdateOrganizationDto,
+  UpdateOrganizationResponse,
+} from '@/routes/organizations/entities/update-organization.dto.entity';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import { RowSchema } from '@/datasources/db/v1/entities/row.entity';
```

Comment:
Nit: I'd add a new line after imports.
```suggestion
import { RowSchema } from '@/datasources/db/v1/entities/row.entity';

```

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/2330/commits/c79d0ad56bc7917eb478754c994bb08fec48a743